app/assets/stylesheets/common.scss

```diff
@@ -1417,7 +1421,7 @@ tr.turn:hover {
     margin-top: $lineheight/2;
   }
 
-  &.deemphasize {
+  &.deleted {
```

Comment:
What's the logic behind keeping this but changing the class it matches?

Reply:
I think the class name is better as "deleted", since that's what has happened to the entry/comment. The associated css (setting a red background) isn't de-emphasizing anything, so it's a bit misleading to keep the original name.

Reply:
Ah I missed that you had added that class in to some things.